src/privatesend/privatesend-client.cpp

```diff
@@ -1504,8 +1505,15 @@ bool CPrivateSendClientSession::CreateDenominated(CAmount nBalanceToDenominate,
     std::vector<CRecipient> vecSend;
     CKeyHolderStorage keyHolderStorageDenom;
 
+    CCoinControl coinControl;
+    // Every input will regure at least this much fees in duffs
```

Comment:
I guess that should mean "require"? 
```suggestion
    // Every input will require at least this much fees in duffs
```